egs/iam/s5/local/chain/align_nnet3_lats.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
```

Comment:
Should we move this script to `steps/nnet3`?

Reply:
See how it differs from steps/nnet3/align_lats.sh.

Reply:
Actually, they are basically the same, however `steps/nnet3/align_lats.sh` was not there at the time.
@aarora8, could you please update the chainali
recipes to use `steps/nnet3/align_lats.sh` instead and remove this script?

Reply:
done, made the change for the pull request.